src/reason-parser/reason_pprint_ast.ml

```diff
@@ -4120,6 +4120,19 @@ let printer = object(self:'self)
     let letPattern = (label ~space:true (atom labelOpener) bindingPattern) in
     (formatTypeConstraint letPattern typeConstraint)
 
+  method formatModuleTypeOfSignatureBinding labelOpener bindingPattern typeConstraint =
+    let letPattern = makeList
+        ~break:IfNeed
+        ~inline:(true, true)
+        [(atom labelOpener);
+         makeList
+           ~pad:(true, false)
+           [bindingPattern; atom ":"]]
+    in
+    makeList
+      ~inline:(true, true)
+      ~break:Always
+      [letPattern; makeList ~wrap:("  ", "") [typeConstraint]]
```

Comment:
```ocaml
makeList
  ~inline:(true, true)
  ~break:IfNeed
  ~postSpace:true
  [letPattern; typeConstraint]
```
I think you want `~postSpace:true` instead of `~wrap:("  ", "")`.
`postSpace` inserts a space after every separator between items in a list.
If the separator is empty (i.e. empty string), it still inserts a simple space.

Reply:
I tried that but `~inline:(true, true)` makes it be 0-based indented... `wrap` here is basically a hack

Reply:
I tried that too. This was the only thing I arrived at that worked 😢 

There's a case where we don't need the `wrap` hack but then if there's a preceding comment if formats like:

```reason
/* https://github.com/facebook/reason/issues/1614 */

    module Event:
    (module type of {
       include ReactEventRe
     })
```